ledger/participant-integration-api/src/main/scala/platform/store/IndexMetadata.scala

```diff
@@ -40,6 +40,7 @@ object IndexMetadata {
     JdbcLedgerDao.readOwner(
       serverRole = ServerRole.ReadIndexMetadata,
       jdbcUrl = jdbcUrl,
+      connectionPoolSize = 2,
```

Comment:
Why 2?

For what it's worth, I would really like numbers that are not `0` or `1` to go into a constant with a good name and ideally a comment explaining why it's set the way it is.

Reply:
Indeed. I lowered it to one and tested it. Thanks for spotting it.